PySubtitle/Helpers/__init__.py

```diff
@@ -488,10 +339,10 @@ def SanitiseSummary(summary : str, movie_name : str = None, max_summary_length :
 
     if len(summary) != original_len:
         logging.info(f"Summary was truncated from {original_len} to {len(summary)} characters")
-    
+
     return summary or None
 
-def FormatErrorMessages(errors : list[SubtitleError]):
+def FormatErrorMessages(errors : List[SubtitleError]):
```

Comment:
Oh, so that's how you do type hints for lists!

Reply:
Heh, yeah, it's totally not obvious with those types. I've just stumbled upon them via experience